android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/MainActivity.java

```diff
@@ -158,7 +158,7 @@ private void showNearbyBeaconsFragment(boolean isDemoMode) {
       }
     } else {
       getFragmentManager().beginTransaction()
-          .setCustomAnimations(R.anim.fade_in_and_slide_up_fragment, R.anim.fade_out_fragment, R.anim.fade_in_activity, R.anim.fade_out_fragment)
+          .setCustomAnimations(R.anim.slide_up_fragment, R.anim.fade_out_fragment, R.anim.fade_in_activity, R.anim.fade_out_fragment)
```

Comment:
Do you still need this line if you're animating afterwards? How does it look without this animation?